media-libs/libilbc/libilbc-1.1.1.ebuild

```diff
@@ -16,7 +16,7 @@ fi
 inherit eutils multilib ${SCM}
 
 DESCRIPTION="Packaged version of iLBC codec from the WebRTC project"
-HOMEPAGE="https://github.com/dekkers/libilbc"
+HOMEPAGE="https://github.com/lu-zero/libilbc"
```

Comment:
Are you sure this is the correct homepage and not just a Gentoo fork? (lu-zero's a Gentoo dev)